src/models/People.js

```diff
@@ -98,7 +98,7 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
       
       //Query available permissions
       let defaultEntityType, defaultEntityTypeData;
-      const defaultEntityTypeSQL = `SELECT entity_types.slug from entity_types LIMIT 1`;
+      const defaultEntityTypeSQL = `SELECT slug FROM entity_types ORDER BY id ASC LIMIT 1`;
```

Comment:
I consider it better practice to be explicit on the table names of each columns, so that it is easier as well to manage when we need to join tables in the future.
```suggestion
      const defaultEntityTypeSQL = `SELECT entity_types.slug FROM entity_types ORDER BY id ASC LIMIT 1`;
```